<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>addShortcutDialog</class>
 <widget class="QDialog" name="addShortcutDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>316</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>500</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>500</width>
    <height>316</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_4">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>16</number>
   </property>
   <property name="topMargin">
    <number>16</number>
   </property>
   <property name="rightMargin">
    <number>40</number>
   </property>
   <property name="bottomMargin">
    <number>24</number>
   </property>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <property name="spacing">
      <number>16</number>
     </property>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>8</number>
       </property>
       <item>
        <widget class="QLabel" name="m_logo">
         <property name="minimumSize">
          <size>
           <width>24</width>
           <height>36</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>24</width>
           <height>36</height>
          </size>
         </property>
         <property name="text">
          <string>TextLabel</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="TitleLabel" name="titleLabel">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>100</width>
           <height>36</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>36</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>24</number>
       </property>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <property name="spacing">
          <number>8</number>
         </property>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="0,0,0">
           <property name="spacing">
            <number>8</number>
           </property>
           <item>
            <widget class="FixLabel" name="label_3">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>60</width>
               <height>36</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>60</width>
               <height>36</height>
              </size>
             </property>
             <property name="text">
              <string>Exec</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="execLineEdit">
             <property name="minimumSize">
              <size>
               <width>160</width>
               <height>36</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>36</height>
              </size>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="openBtn">
             <property name="minimumSize">
              <size>
               <width>84</width>
               <height>36</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>84</width>
               <height>36</height>
              </size>
             </property>
             <property name="focusPolicy">
              <enum>Qt::NoFocus</enum>
             </property>
             <property name="text">
              <string>Open</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_6" stretch="0,0">
           <property name="spacing">
            <number>8</number>
           </property>
           <item>
            <widget class="FixLabel" name="label_2">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>60</width>
               <height>36</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>60</width>
               <height>36</height>
              </size>
             </property>
             <property name="text">
              <string>Name</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="nameLineEdit">
             <property name="minimumSize">
              <size>
               <width>16</width>
               <height>36</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>36</height>
              </size>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="FixLabel" name="label">
             <property name="minimumSize">
              <size>
               <width>60</width>
               <height>36</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>60</width>
               <height>36</height>
              </size>
             </property>
             <property name="text">
              <string>Key</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QFrame" name="wFrame">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>48</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>48</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <property name="spacing">
         <number>8</number>
        </property>
        <property name="leftMargin">
         <number>24</number>
        </property>
        <item>
         <spacer name="horizontalSpacer_5">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Fixed</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>60</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QFrame" name="frame">
          <property name="minimumSize">
           <size>
            <width>326</width>
            <height>40</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>40</height>
           </size>
          </property>
          <property name="frameShape">
           <enum>QFrame::Box</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <property name="lineWidth">
           <number>0</number>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_9">
           <property name="spacing">
            <number>16</number>
           </property>
           <property name="leftMargin">
            <number>0</number>
           </property>
           <property name="topMargin">
            <number>0</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_8">
             <property name="spacing">
              <number>4</number>
             </property>
             <item>
              <widget class="QLabel" name="noteLabel">
               <property name="minimumSize">
                <size>
                 <width>24</width>
                 <height>36</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>24</width>
                 <height>36</height>
                </size>
               </property>
               <property name="text">
                <string>TextLabel</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="FixLabel" name="label_4">
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>36</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>16777215</width>
                 <height>36</height>
                </size>
               </property>
               <property name="text">
                <string>TextLabel</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_7">
     <property name="spacing">
      <number>8</number>
     </property>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="cancelBtn">
       <property name="minimumSize">
        <size>
         <width>120</width>
         <height>36</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>120</width>
         <height>36</height>
        </size>
       </property>
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="certainBtn">
       <property name="minimumSize">
        <size>
         <width>120</width>
         <height>36</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>120</width>
         <height>36</height>
        </size>
       </property>
       <property name="text">
        <string>Save</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>TitleLabel</class>
   <extends>QLabel</extends>
   <header>Label/titlelabel.h</header>
  </customwidget>
  <customwidget>
   <class>FixLabel</class>
   <extends>QLabel</extends>
   <header>Label/fixlabel.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
